content/spin/javascript-components.md

```diff
@@ -380,6 +383,18 @@ export const handleRequest: HandleRequest = async function(request: HttpRequest)
 }
 ```
 
+## Storing Data in the Spin Key-Value Store
+
+Spin has a key-value store built in. For information about using it from Rust, see [the key-value store tutorial](kv-store-tutorial).
+
+## Storing Data in SQLite Relational Databases
+
+For more information about using SQLite relational databases from TypeScript/Javascript, see [SQLite storage](sqlite-api-guide).
+
+## Storing Data in MySQL and PostgreSQL Relational Databases
```

Comment:
Not supported in TS/JS as far as I know

Reply:
Left this in for the sake of the "However, you can use hosted relational database services that are accessible over HTTP" at the resolving link; in case someone wants to user ElephantSQL or something over HTTP via TS/JS.